Toolset/libraries/revbackscriptlibrary.livecodescript

```diff
@@ -3561,3 +3563,65 @@ private function utilityPathIsCaseSensitive pPath
   
   return tIsCaseSensitive
 end utilityPathIsCaseSensitive
+
+constant kHandleSize = 5
+
+on ideMouseMove pX, pY, pTarget
```

Comment:
@montegoulding Could you please add some comments here that explain what this handler is for?